### DISCUSSION TITLE
Does buying WAHA Plus mean I have to buy it again every month to use it?

### DISCUSSION THREAD

**Discussion by mehmetcoskun** [devlikeapro/waha#58] [Q&A]
Does buying WAHA Plus mean I have to buy it again every month to use it? the instructions on the website say this but I want to make sure :)

Thank You.

**Comment by allburov** [comment]
Hi!
First of all, we don't sell waha, it's not a purchase, it's a donation. Almost everything for a small bot WAHA provides in Core version, Plus is just our way to say Thank you!

No, you don't have, the installed version has no license check.

But time to time WhatsApp adds some backward incompatible changes, so you'll need to update WAHA in order to make it work again.
You can assume that it happens time to time, like every 3-6 months - here you'll have to donate one lore time to get new api keys.

**Reply by mehmetcoskun** [reply]
Well, I don't know if patreon invoices the payments? If not, how else can we pay?

**Reply by allburov** [reply]
Yep, you can donate either on [Boosty](https://boosty.to/wa-http-api) or on Patreon. 
We prefer Boosty :)

**Comment by mehmetcoskun** [comment]
I bought from Boosty but I didn't get a password. where can I get the password?

**Reply by allburov** [reply]
Hi, thank you for supporting the project!
To download WAHA PLUS image - please use the key in the latest post.
Please let us know if you have any questions!

https://boosty.to/wa-http-api/posts/08b97d1d-a849-42ba-8430-a5741ae96f3a?share=post_link

**Comment by bhishmakajaria** [comment]
Hello @allburov I have donated you on Patreon. Can you please share the code as you have mentioned in your discussion if the code is needed it can be shared.

Awaiting you prompt response. My mail id is kajaria.bhishma@gmail.com

**Reply by allburov** [reply]
Hi! @bhishmakajaria I've invited you to the private repo https://github.com/devlikeapro/whatsapp-http-api-plus

**Reply by bhishmakajaria** [reply]
Thank you so much...

**Reply by John21DE** [reply]
Hi.
I also donated the 19 Dollars as I think, this is worth every penny.
On the other hand, I also did not receive a Password, also no Post, Text/SMS message or similar.
So also reading everything carefully and finally following the in the FAQ  "Read more about how to get PASSWORD for Plus Version →" I am just sent back to sign up for the Plus Donation (which is already done).

In the last part of the "Get Plus" documentation it just says under "Install Plus" ....
[..] After you get the password, [...]
But nowhere is a hint on how to get it.

So, please advise and complete the documentation.
Thx.
John21

**Reply by allburov** [reply]
Hi! Sorry for the confusion, you can find the actual password in the latest post on Boosty or Patreon!
We'll think about how to improve the flow and make it simpler, I see that people don't get where to find the key :(

**Reply by John21DE** [reply]
I already read "the latest post" on Boosty, but to be honest, I cannot see a single post. Seems I am the only person in the whole space there...

**Reply by allburov** [reply]
I think if you login and refresh the page you'll see it 
I've sent the password to DM in Boosty
Sorry for the inconvenience!

**Comment by TitoBahe** [comment]
Hello @allburov, you said before that from time to time WhatsApp adds some backward incompatible changes... So do I need to regenerate the API key periodically after making a donation? My session needs to run 24/7. What happens to the session in the meantime, before I get another image from Waha-Plus?

**Reply by devlikepro** [reply]
> 24/7

I don't think even official API can offer you this 99.9% uptime :) 

Update it time to time and you'll be fine (ideally every month), we usually get backward incompatible changes and prepare the project for them early then WA roll outs it to all users.


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by TitoBahe** [reply]
So I need to make a donation every month if I want to get the updated version of WAHA every month?

**Reply by devlikepro** [reply]
Correct

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by arlyaraujo** [comment]
hello @allburov 

I have some questions.

When I go to pay for the booster or Prono, it says that the fee is monthly. Is the fee monthly or fixed?
Do I have access to updates for WahaPro? Or will I have to pay for each update?
Can I install it on more than one server?

**Reply by devlikepro** [reply]
Hi! It's kinda mixed, not "every month", but rather "every 3-6 months" fees
https://waha.devlike.pro/docs/overview/faq/#i-want-to-purchase-waha-plus-waha-pro-is-it-one-time-purchase

Yes, you can install it on any number of servers with any numbers of WhatsApp accounts associated, no limits there

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by suzukiahmed** [comment]
Whether can i install waha pro in my multiple vps using a single waha pro donation?

**Comment by grupoprogestionchile** [comment]
Fm

**Comment by grupoprogestionchile** [comment]
Fm
